packages/components/app/styles/components/code-block/index.scss

```diff
@@ -0,0 +1,209 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// CODE-BLOCK
+//
+
+
+// Note: "theme" contains just color variables and syntax highlighting styles
+@import "theme";
+
+// DIMENSIONS
+$code-block-line-numbers-width: 49px; // 3em = 49px
+$code-block-code-padding: 16px;
+
+// CODE-BLOCK PARENT/WRAPPER
+
+.hds-code-block {
+  position: relative;
+  overflow: hidden; // So header corners don't poke out from container
+  color: var(--code-block-color-foreground-primary);
+  font-family: var(--hds-typography-code-200);
+  background-color: var(--code-block-color-surface-primary);
+  // Note: may change border-radius to be toggled by an "@isRounded" property
+  border-radius: 6px;
+
+  pre,
+  code {
+    line-height: var(--token-typography-body-200-line-height); // = 1.4286 | 20px
```

Comment:
This is the value in question, the 18px line-height seemed and  too tight and we've received [feedback](https://hashicorp.atlassian.net/browse/HDS-2260) on this already. I had proposed updating to 20px which gives a bit more spacing, but we need to decide whether to make this change holistically across the system and what the best method is. A couple of proposals to consider:

1. Update the line height of just the `code / 200` token to provide a bit more space, though we'll need to assess what other contexts it is used in. This would match the line height of `body / 200`.
2. Update _all_ of the code styles to match their body equivalents, which may be a more holistic solution (but hard to tell without knowing the reason for the underlying decision)
3. Use a different value for the line height in the Ember component, which to match would result in the Figma style needing to be detached (as font-size and line-height cannot be decoupled in Figma)
4. Not do anything and leave as is for now. Address this ticket in the future.

For numbers 1 and 2 there may be a number of impacts across consumers and within our own components that may be hard to gauge. 

I don't have a strong opinion either way, but it would be nice to either address this feedback now, and if we aren't going to make the change, communicate that and icebox it.

cc. @didoo and @heatherlarsen

Reply:
@jorytindall 

> but hard to tell without knowing the reason for the underlying decision

Can you clarify what you mean by this?

> Use a different value for the line height in the Ember component, which to match would result in the Figma style needing to be detached (as font-size and line-height cannot be decoupled in Figma)

Is the proposal for this option to change the style only for the CodeBlock component? If so, then I'd think that neither the ember component nor the Figma component would use an existing semantic style and would use a custom (possibly local) style instead. So it wouldn't necessarily be that the Figma component is "detached", but that it uses a new style specific to solving the needs of the use case, which we do every so often when it's needed, e.g., form controls. I'm not advocating for this option over the others; just try to reframe it. 

> For numbers 1 and 2 there may be a number of impacts across consumers and within our own components that may be hard to gauge.

Did you create visual examples of how a holistic change would impact other components in the system using this style?